ttm/releaser.py

```diff
@@ -33,24 +33,24 @@ def release(self, project, force=False):
         # not overwriting
         if new_snapshot == testing_snapshot:
             self.logger.debug('no change in snapshot version')
-            return
+            return None
 
         if testing_snapshot != self.get_status('failed') and testing_snapshot != self.get_status('published'):
             self.logger.debug('Snapshot {} is still in progress'.format(testing_snapshot))
-            return
+            return QAResult.inprogress
 
         self.logger.info('testing snapshot %s', testing_snapshot)
         self.logger.debug('new snapshot %s', new_snapshot)
 
         if not self.is_snapshotable():
             self.logger.debug('not snapshotable')
-            return
+            return QAResult.failed
```

Comment:
Should be `None` - it just can't tell what's going on at this point.

Reply:
Well, possibly we need to split this further - but it's false if the products don't build or are oversized